### PR TITLE
Deck, remove --cookie-secret-file default value

### LINKED ISSUE #21795: Deprecate `--cookie-secret-file` handling in deck
**What should be cleaned up or changed**:
remove this handling after 2019-10-31

**Provide any links for context**:
https://github.com/kubernetes/test-infra/blob/176e4b678ce11d0c4640d818d57034e9ae523403/prow/cmd/deck/main.go#L143-L146

### PR DESCRIPTION
**Pull request by matthyx**
fix #21795

### CODE REVIEW

File: prow/ANNOUNCEMENTS.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## New features
 
 New features added to each component:
+  - *April 14, 2021* Deck remove default value for `--cookie-secret-file`.
```
Comment (cjwagner) [inline prow/ANNOUNCEMENTS.md:6]: This should go under the breaking changes section, not the new features section.

Comment (matthyx) [reply prow/ANNOUNCEMENTS.md]: oops, thanks Cole!

**Review by cjwagner — APPROVED** [review]
Thanks Matthias!

**Pull request merged**